howso/attributes.amlg

```diff
@@ -1,161 +1,158 @@
+			;
+			;		'rate_min': list of number, if specified, ensures that the rate (the difference quotient, the discrete version
+			;					of derivative) for this feature won't be less than the value provided. A null value means no min boundary.
+			;					The value must be in epoch format for the time feature. The length of the list must match the number of
+			;					derivatives as specified by 'order'. Only applicable when time series type is set to 'rate'. Example: [0.1]
+			;
+			;		'rate_max': list of number, if specified, ensures that the rate (the difference quotient, the discrete version
+			;					of derivative) for this feature won't be more than the value provided. A null value means no max boundary.
+			;					The value must be in epoch format for the time feature. The length of the list must match the number of
+			;					derivatives as specified by 'order'. Only applicable when time series type is set to 'rate'. Example: [0.8]
+			;
+			;		'delta_min': list of number, if specified, ensures that the smallest difference between features values is not smaller
+			;					than this specified value. A null value means no min boundary. The length of the list must match the number of
+			;					derivatives as specified by 'order'. Only applicable when time series type is set to 'delta'. Example: [0.2]
+			;
+			;		'delta_max': list of number, if specified, ensures that the largest difference between feature values is not larger than
+			;					this specified value. A null value means no max boundary. The length of the list must match the number of
+			;					derivatives as specified by 'order'. Only applicable when time series type is set to 'delta'. Example: [2.0]
+			;
+			;		'series_has_terminators': boolean, when true, requires that the model identify and learn values that explicitly denote
+			;								the end of a series. Only applicable to id features for a series. Default is false
+			;
+			;		'stop_on_terminator': boolean, when true, requires that a series ends on a terminator value.
+			;							Only applicable to id features for a series. Default is false
+			;
+			;		'universal': boolean, applicable only to the time_feature, controls whether future values of independent time series are considered.
+			;					When false, time_feature is not universal and allows using future data from other series in decisions;
+			;					this is applicable when the time is not globally relevant and is independent for each time series.
+			;					When true, universally excludes using any data with from the future from all series;
+			;					this is applicable when time is globally relevant and there are events that may affect all time series.
+			;					If there is any possibility of global relevancy of time, it is generally recommended to set this value to true, which is the default.
 			feature_attributes (assoc)
 			;{type "boolean"}
+			;flag, default to true. create time series feature attributes if necessary
```

Comment:
remove 'flag,', it's type is a boolean, so that's redundant

Reply:
Will change pending discussion in Teams

Reply:
Per discussion in Teams, will be deferred for this scope of work.